packages/@react-aria/virtualizer/src/VirtualizerItem.tsx

```diff
@@ -88,15 +88,6 @@ export function layoutInfoToStyle(layoutInfo: LayoutInfo, dir: Direction, parent
     ...rectStyles
   };
 
-  if (layoutInfo.isSticky && !parent?.allowOverflow) {
```

Comment:
haven't found any place that this code was affecting other than to make our S2 Tables empty and spinners not centered